src/metabase/query_processor/middleware/annotate.clj

```diff
@@ -637,7 +715,7 @@
 (defn add-column-info
   "Middleware for adding type information about the columns in the query results (the `:cols` key)."
   [{query-type :type, :as query
-    {:keys [:metadata/model-metadata :alias/escaped->original]} :info} rff]
+    {:keys [card-entity-id :metadata/model-metadata :alias/escaped->original]} :info} rff]
```

Comment:
I find it easier to recognize names if they look like symbols (especially with color coding):
```suggestion
    {:keys [card-entity-id metadata/model-metadata alias/escaped->original]} :info} rff]
```